### ISSUE TITLE
Missing security policy for Preview

### ISSUE THREAD

**Issue by JBouwer** [JBouwer/boost-quickbook-support#3]
From the console during debugging:
```JBouwer.boost-quickbook-support created a webview without a content security policy: https://aka.ms/vscode-webview-missing-csp```

**Comment by JBouwer** [comment]
The [MS documentation](https://code.visualstudio.com/api/extension-guides/webview#content-security-policy) suggests an easy fix, and it seems to work _sometimes_ - but not _all of the time_.

The catch seems to be that the html content trigger the message - with or without a security policy.
See commit 64a29e9b2fe15ad7232db69029a0697fb6ff9e67.

**Comment by JBouwer** [comment]
I also tried to ass the _additional directives_ - as listed in [Google Web Fundamentals - Content Security Policy](https://developers.google.com/web/fundamentals/security/csp/):

```
let strProcessedContents = self.processPreview( strContents, "default-src vscode-resource:; "
                                              + "base-uri vscode-resource:; "
                                              + "form-action vscode-resource:; "
                                              + "frame-ancestors vscode-resource:; "
                                              + "plugin-types vscode-resource:; "
                                              + "report-uri vscode-resource:; "
                                              + "sandbox vscode-resource:; "
                                              );
```
... but to no avail.

**Comment by JBouwer** [comment]
This was fixed - along with #2 and released as v0.0.4.

> Note that the:  
> `JBouwer.boost-quickbook-support created a webview without a content security policy: https://aka.ms/vscode-webview-missing-csp`
> warning message remains... At this this time I have no idea how to get rid of it.
